src/org/parosproxy/paros/control/Control.java

```diff
@@ -504,19 +505,31 @@ public void createAndOpenUntitledDb() throws ClassNotFoundException, Exception {
 
 	@Override
 	public void sessionOpened(File file, Exception e) {
-		getExtensionLoader().databaseOpen(model.getDb());
-		getExtensionLoader().sessionChangedAllPlugin(model.getSession());
+		notifyExtensionsSessionChanged(e);
+
 		if (lastCallback != null) {
 			lastCallback.sessionOpened(file, e);
 			lastCallback = null;
 		}
 		
 	}
 
+	/**
+	 * Notifies the extensions that the session changed, iff the given exception is {@code null}.

```

Comment:
Little typo on `iff`

Reply:
That was on purpose for "if and only if" but not necessary?

Reply:
Huh, I've never seen that before. I'm sure it's "technically" correct. I think it's probably clear enough as "if" but if not then it should probably actually say "if and only if".

Reply:
OK, changed to "if".